.github/workflows/publish-site.yml

```diff
@@ -0,0 +1,27 @@
+name: Website Build
+on:
+  push:
+    branches:
+      - 'feature/heavy-wip-website'
+  pull_request:
+    # none
+jobs:
+  Build:
+    runs-on: windows-latest
```

Comment:
Does this have to be windows? They are usually harder to aquire and given we end up in a queue always already, Ubuntu would be preferred I think.

Reply:
Let's see...
```suggestion
    runs-on: ubuntu-latest
```